src/overview/container.js

```diff
@@ -214,15 +278,35 @@ class OverviewContainer extends Component {
             !wereAnyClicked(this.furtherTagRefs)
         ) {
             this.props.resetActiveTagIndex()
+            this.props.resetFilterPopup()
         }
     }
 
     render() {
+        const {
+            shouldDisplayDomainFilterPopup,
+            shouldDisplayTagFilterPopup,
+            filterTags,
+            filterDomains,
+        } = this.props
+
         return (
             <Overview
                 {...this.props}
                 setInputRef={this.setInputRef}
                 onInputChange={this.props.handleInputChange}
+                tagFilterManager={this.renderTagsFilter(
```

Comment:
These methods should not have `this.props` passed in as args - they are instance methods, so have access to `this`.